src/runtime/config/configuration-qemu-coco-dev.toml.in

```diff
@@ -713,3 +713,7 @@ create_container_timeout = @DEFCREATECONTAINERTIMEOUT@
 # to the hypervisor.
 # (default: /run/kata-containers/dans)
 dan_conf = "@DEFDANCONF@"
+
+# Enforce guest pull. This instructs the runtime to communicate to the agent via annotations that
```

Comment:
Typo in the commit message `s/runtim/runtime/`

Reply:
Fixed.